examples/AccountManagement/ApproveMerchantCenterLink.php

```diff
@@ -0,0 +1,197 @@
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $merchantCenterAccountId the Merchant Center account ID
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $merchantCenterAccountId
+    ) {
+        // Lists all merchant links of the specified customer ID.
+        $merchantCenterLinkServiceClient = $googleAdsClient->getMerchantCenterLinkServiceClient();
+        $response = $merchantCenterLinkServiceClient->listMerchantCenterLinks($customerId);
+        printf(
+            "%d Merchant Center link(s) found with the following details:%s",
+            $response->getMerchantCenterLinks()->count(),
+            PHP_EOL
+        );
+        foreach ($response->getMerchantCenterLinks() as $merchantCenterLink) {
+            /** @var MerchantCenterLink $merchantCenterLink */
+            printf(
+                "Link '%s' has status '%s'.%s",
+                $merchantCenterLink->getResourceName(),
+                MerchantCenterLinkStatus::name($merchantCenterLink->getStatus()),
+                PHP_EOL
+            );
+            // Approves a pending link request for a Google Ads account with the specified customer
+            // ID from a Merchant Center account with the specified merchant center account ID.
+            if (
+                $merchantCenterLink->getIdUnwrapped() === $merchantCenterAccountId
+                && $merchantCenterLink->getStatus() === MerchantCenterLinkStatus::PENDING
+            ) {
+                // Updates the status of Merchant Center link to 'ENABLED' to approve the link.
+                self::updateMerchantCenterLinkStatus(
+                    $merchantCenterLinkServiceClient,
+                    $customerId,
+                    $merchantCenterLink,
+                    MerchantCenterLinkStatus::ENABLED
+                );
+                // There is only one MerchantCenterLink object for a given Google Ads account and
+                // Merchant Center account, so we can break early.
+                break;
+            }
+        }
+    }
+
+    /**
+     * Updates the status of a Merchant Center link with a specified merchant center link status.
```

Comment:
Shall we use "Merchant Center" instead of "merchant center" for consistency?

Reply:
Yes, we should.